app/src/main/AndroidManifest.xml

```diff
@@ -106,7 +106,6 @@
       android:screenOrientation="portrait" />
     <activity
       android:name=".topic.TopicActivity"
-      android:screenOrientation="portrait"
```

Comment:
Instead of this, you will need to make `ConceptCardFragmentTestActivity` work for landscape mode. You don't need to do anything for TopicActivity or related files.